src/server/injectors/script-injector.ts

```diff
@@ -164,7 +166,7 @@ export default class ScriptInjector implements KrasInjector {
           $server: this.core,
           ...extended,
         };
-        const res = handler(ctx, req, builder);
+        const res = handler.call(this, ctx, req, builder);
```

Comment:
Sure we can leave this, but we could (maybe in addition?) also add `$options` and `$config` to the context `ctx`.

Reply:
Your suggestion is better and has been updated!